draft-pda-protocol.md

```diff
@@ -419,12 +418,15 @@ Duration uint64; /* Number of seconds elapsed between two instants */
 Time uint64; /* seconds elapsed since start of UNIX epoch */
 ~~~
 
-* `uuid`: A unique sequence of bytes used  to ensure that two otherwise
+* `uuid`: A unique sequence of bytes used to ensure that two otherwise
```

Comment:
I wouldn't call this "uuid" because then some IETF nitpicker will complain that it's not actually a UUID.

Maybe ```task_nonce```

Reply:
I agree, but I think `struct Param` will go away altogether when someone (perhaps me, later this week) addresses #104, and we will be left with nothing but `task_id`. So I prefer to leave this text alone until it gets deleted in that PR (except, I guess, to delete this extra space).